domains/auth/Auth.API/API/Orchestrator/Orchestrator.cs

```diff
@@ -0,0 +1,46 @@
+using API.Configuration;
+using API.Models;
+using API.Services.OidcProviders;
+using Microsoft.Extensions.Options;
+
+namespace API.Orchestrator;
+
+public class Orchestrator : IOrchestrator
```

Comment:
I think the name `Orchestrator` is a bit fluffy and i am not sure what an `Orcestrator` is supposed to do.
Maybe i can get an introduction IRL?